platform/binance/transaction.go

```diff
@@ -3,198 +3,20 @@ package binance
 import (
 	"github.com/trustwallet/blockatlas/coin"
 	"github.com/trustwallet/blockatlas/pkg/blockatlas"
-	"github.com/trustwallet/blockatlas/pkg/numbers"
-	"strconv"
-	"strings"
-	"sync"
 )
 
-const emptyToken = ""
-
 func (p *Platform) GetTxsByAddress(address string) (blockatlas.TxPage, error) {
-	explorerResponse, err := p.GetTokenTxsByAddress(address, emptyToken)
+	txsFromClient, err := p.client.FetchTransactionsByAddressAndAssetID(address, coin.Binance().Symbol)
```

Comment:
FetchTransactionsByAddressAndAssetID - but params `address` and `coin.Binance().Symbol`
Symbol != assetID?

Reply:
Token ID will be better 👍